src/main/java/com/google/devtools/build/lib/runtime/commands/BuildCommand.java

```diff
@@ -52,7 +53,8 @@
       LoadingOptions.class,
       KeepGoingOption.class,
       LoadingPhaseThreadsOption.class,
-      BuildEventProtocolOptions.class
+      BuildEventProtocolOptions.class,
+      RemoteOptions.class,
```

Comment:
Since we have moved the `RemoteDownloaderConfig` from `RemoteOptions` to `RepositoryOptions`, can you check whether adding `RemoteOptions.class` here and `//src/main/java/com/google/devtools/build/lib/remote/options` in BUILD files are necessary?